apiserver/facades/client/secrets/secrets.go

```diff
@@ -50,7 +50,7 @@ func (s *SecretsAPI) checkCanWrite(ctx context.Context) error {
 }
 
 func (s *SecretsAPI) checkCanAdmin(ctx context.Context) error {
-	isAdmin, err := common.HasModelAdmin(ctx, s.authorizer, names.NewControllerTag(s.controllerUUID), names.NewModelTag(s.modelUUID))
+	isAdmin, err := model.HasModelAdmin(ctx, s.authorizer, names.NewControllerTag(s.controllerUUID), names.NewModelTag(s.modelUUID))
```

Comment:
No biggie but `model` should probably be `commonmodel` here like everywhere else to make it clear thats where its coming from.

Reply:
Fair